config/environments/development.rb

```diff
@@ -4,7 +4,7 @@
   # In the development environment your application's code is reloaded on
   # every request. This slows down response time but is perfect for development
   # since you don't have to restart the web server when you make code changes.
-  config.cache_classes = false
+ config.cache_classes = false
```

Comment:
Can we include some type of linter workflow so that these formatting changes get standardized/don't distract from core logic changes? Might be out of scope for this specific pull request, but the sooner we introduce it the easier our lives will get. You have more experience with Ruby than me, so I'm not sure what the best industry practices are.

Reply:
Yep, linters are very common in ruby projects, we can add that as an issue for next week's sprint